dev-quick-start.md

```diff
@@ -0,0 +1,197 @@
+    ✔ GitHub account (for forking and contributing)
+
+
+
+## Repository
+The K8ssandra GitHub repository resides [here](https://github.com/k8ssandra/k8ssandra).  
+
+ 
+
+Once at the K8ssandra repository, click on the **Fork** button (top right corner of screen). This will fork the K8ssandra to your local repository.
+
+Using Git, you can now pull the K8ssandra source code and documentation to your local machine.  Some developers choose to do this via the GitHub plugin-in in an IDE or from a command line.
+
+Perform a clone.  Be sure to insert your GitHub repository name followed by k8ssandra.git.
+      
+>git clone https://github.com/your-github-repo-name/k8ssandra.git
+   
+Now you are ready to explore the K8ssandra project.
+
+### Project composition
+The K8ssandra project has a few important folders of interest.
+
+* **Charts** - contains the Helm-based charts categorized by sub-chart.
+* **Docs** - contains the layouts, & content for the [k8ssandra.io](https://k8ssandra.io) site.  
+Checkout the docs readme for more detail.
+* **Scripts** - contains a collection of useful scripts that developers can build upon or just use directory when working on K8ssandra. Also included are documentation scripts for K8ssandra docs management.
+* **Tests** - contains the set of unit, integration, and end-to-end testing for K8ssandra.
+
+
+## Testing
+Within the test directory are folders for managing and executing tests at the `unit`, `integration`, and `e2e` test levels.
+
+As a working example, using a command line, navigate to the `./k8ssandra/tests/unit` directory.
+
+Issue the command
+
+> go test -v
+
+Once complete, you should see something like the following:
+
+>Ran 145 of 145 Specs in 71.451 seconds
+>
+>SUCCESS! -- 145 Passed | 0 Failed | 0 Pending | 0 Skipped
+>
+> --- PASS: TestTemplateUnitTests (71.47s)
+>PASS
+>
+>ok      github.com/k8ssandra/k8ssandra/tests/unit       71.557s
+
+## Troubleshooting advice
+This section will be updated over time as K8ssandra grows.  Take a look to enhance your K8ssandra experience. 
+
+### Common errors
+
+You may experience a `missing in charts/ directory` error message.  
+
+If so, you can 
+utilize a K8ssandra script: `./scripts/update-helm-deps.sh`. This script assists with updating dependencies for each chart in an appropriate order.  
+
+Be sure to run this script so the `./charts` folder is properly located. 
+
+
+
+  
+### Collecting useful information 
+
+So you have an error after editing a K8ssandra configuration, or you want to inspect some things as you learn.  There are some useful commands that come in handy when needing to dig a bit deeper.  The examples assume you are using a k8ssandra namespace, but this can be adjusted as needed.
+
+Issue the following `kubectl` command to view the `Cassandra` logs.  Replace *<pod-with-cassandra>* with an actual pod instance name having the `Cassandra` container.
+
+> kubectl logs pod/<pod-with-cassandra> cassandra -n k8ssandra
```

Comment:
This is incorrect. It should be `kubectl logs <cassandra-pod> -c cassandra`

Reply:
Good catch!